Library/Formula/ghc.rb

```diff
@@ -10,163 +12,91 @@ class Ghc < Formula
     sha256 "ad654abb4c2459b6cc764f275d5b6b141669e366f5051c58d871e54cb71a250c" => :mountain_lion
   end
 
-  option "32-bit"
```

Comment:
GHC 7.6.3 or newer is needed to bootstrap 7.10. The latest 32-bit bindist is 7.4.1. So the option has been a no-op for a while. While one could build a 32-bit intermediate 7.6.3, then build 7.10 with that, I thought that might be better left for something like tigerbrew to try.